src/experimenting.py

```diff
@@ -0,0 +1,79 @@
+import numpy as np
+import pandas as pd
+from tqdm.notebook import tqdm
+import os.path
+
+
+class ExperimentPerformer(object):
```

Comment:
A bit of description in docstring would be nice.

Reply:
Sure